### DISCUSSION TITLE
Please add a way to get username of user who last modified the cloud var with given name

### DISCUSSION THREAD

**Discussion by KSKgames** [CubeyTheCube/scratchclient#39] [Ideas]
The title /\ /\ /\
![image](https://user-images.githubusercontent.com/52801271/145687166-d931b1ce-9d27-411b-a83d-1c582cf51584.png)

**Comment by Quantum-Codes** [comment]
That might be hard as there are actions like renaming the cloud variable. What should it do then? Show the last value of the "named" variable?

**Reply by KSKgames** [reply]
"What should it do then? Show the last value of the "named" variable?"

That would be great :)

**Reply by Quantum-Codes** [reply]
Ok, let the creator of this package see this. I will try making one
